### ISSUE TITLE
Are the graphs displaying active cases or new cases?

### ISSUE THREAD

**Issue by HeatherHMorgan** [dllh/kcs-tracing.org#23]
What does each bar of the bar graph represent? Does a bar represent the number of cases that *began* on a certain date? Or does a bar represent the total number of cases that are *active* on that date using a decision rule about how long a case stays active (e.g., ten days from the earlier of the onset of symptoms / positve test date)?

**Comment by dllh** [comment]
They're displaying active -- the earlier of the two dates we collect. I hadn't recalled the 10-day rule but should be able to implement that if that's the correct metric.

**Comment by HeatherHMorgan** [comment]
We can leave the logic as is --- that is totally okay with me, but I think we should change the title from "active cases" to "new cases" to align with the way KCHD uses "new cases" and the way KCS uses "active cases."

**Comment by dllh** [comment]
Cool, I've updated the label on the charts to reflect that we're showing new cases. While I was at it, I also updated the submission process so that it also records "new case (which is the earliest of symptomatic or positive test date) plus 10 days" as "active case." So if we did want to show both pieces of data in the charts or to have separate charts or something in the future, it's fairly easy to do so. 

I'll close this issue for now, but if we'd like to revisit, we can reopen.
